src/style.css

```diff
@@ -125,11 +125,16 @@ input[type="submit"] {
   justify-content: center;
 }
 
-.padding-bottom {
+.padding-top-bottom {
   padding-bottom: 30px;
+  padding-top: 30px;
+}
+
+.extra-padding-top {
+  padding-top: 30px;
 }
 
-.padding-top {
+.extra-padding-bottom {
   padding-top: 30px;
```

Comment:
Are you sure the class name should be extra-padding-**bottom**, check your css property.

Reply:
fixed. thanks!